main.py

```diff
@@ -1,7 +1,35 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
```

Comment:
The return type annotation for the `main` function is `QuerySet`, which is correct if you are returning a queryset. Ensure that the `main_actors` variable is indeed a `QuerySet` object, which it should be, given the use of `filter` and `order_by` methods.